CodingSeb.ExpressionEvaluator/ExpressionEvaluator.cs

```diff
@@ -4692,7 +4690,14 @@ public partial class MethodsGroupEncaps
 
     public partial class BubbleExceptionContainer
     {
-        public Exception Exception { get; set; }
+        public BubbleExceptionContainer(Exception exception)
+        {
+            _dispatchInfo = ExceptionDispatchInfo.Capture(exception);
+        }
+
+        private readonly ExceptionDispatchInfo _dispatchInfo;
+
+        public void Throw() => _dispatchInfo.Throw();
```

Comment:
`BubbleExceptionContainer` is kept to minimize source breaking changes. `Exception` property is removed, but it can be returned